src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/BulkIndexationIntegration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch\IndexConfiguration;
```

Comment:
here you test only the ES client, which means you should put it in the AkeneoElasticSearchBundle

but I'm even wondering if this test is useful or not
shouldn't have instead tests to check that when we save products, they are indexed in ES?